.github/workflows/yum-arm.yml

```diff
@@ -44,3 +44,13 @@ jobs:
       - name: Check Package Size
         run: |
           td-agent/yum/pkgsize-test.sh ${{ matrix.rake-job }} aarch64
+      - name: Binstubs Test
+        run: |
+          mkdir -p .bundle
+          docker run \
+          --rm \
+          --tty \
+          --volume ${PWD}:/fluentd:ro \
+          --env CENTOS_STREAM=false \
+          ${{ matrix.test-docker-image }} \
```

Comment:
This workflow still uses x86_64 images for centos and amazonlinux 2.
Should we use `arm64v8/` prefixed images like as apt-arm workflow?

WDYT?

Reply:
Oh, I've overlooked it, thanks.

Reply:
resolved.